test/fixtures/output/stream-share/delete-consumer-share-multiple-fail.golden

```diff
@@ -0,0 +1,5 @@
+Error: consumer share not found: "ss-12346"
+
+Suggestions:
+    Use the `--skip-invalid` flag to suppress this error and delete the remaining consumer share entered.
```

Comment:
Should this be plural? Also are we sure we want this flag in the first place?
```suggestion
    Use the `--skip-invalid` flag to suppress this error and delete the remaining consumer shares entered.
```

Reply:
No, we don't need this flag. So I'll remove this. Kind of moot now, but to answer the first question: the message was set up to be singular or plural based on the count of remaining valid entries.